airbyte-cron/src/main/java/io/airbyte/cron/MicronautCronRunner.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cron;
+
+import io.airbyte.cron.selfhealing.Temporal;
+import io.micronaut.runtime.Micronaut;
+import javax.inject.Inject;
+
+/**
+ * Micronaut server responsible of running scheduled method. The methods need to be separated in
+ * Bean based on what they are cleaning and contain a method annotated with `@Scheduled`
+ *
+ * Injected object looks unused but they are not
+ */
+public class MicronautCronRunner {
+
+  @Inject
+  Temporal temporal;
```

Comment:
If this is just to ensure that the bean gets created, you could also do this through an [ApplicationListener](https://docs.micronaut.io/latest/guide/#serverEvents) or by configuring Micronaut to do [eager initialization of Singletons](https://docs.micronaut.io/latest/guide/#eagerInit).

Reply:
Done